readme.md

```diff
@@ -35,7 +35,7 @@ You can configure the Stripe.net package to use your secret API key in one of tw
 a) In your application initialization, set your API key (only once once during startup):
 
 ```csharp
-StripeConfiguration.SetApiKey("[your api key here]");
+StripeConfiguration.ApiKey = "[your api key here]";
```

Comment:
My intention was to avoid a breaking change by keeping the old methods around, but marking them as `Obsolete` to encourage users to upgrade to the new way of setting these values, and actually remove the methods in a future major version. This would make the upgrade a bit less painful for users.

I don't feel too strongly about it, if you'd rather we drop the old methods immediately that's fine by me.

Reply:
Okay I think that's fine and a nicer experience for sure